April23/news.md

```diff
@@ -0,0 +1,29 @@
+# News
+
+## FlashFest organised for April Fools’ snapshot
+
+This month of course comes with April Fools' Day, and that means a Minecraft
+April Fools' update! In this year's offering we saw a feature in which players
+regularly vote on "proposals", each of which would make a small change to the
+game, ranging from changing the colour of grass to... turning the game French?
+C'est bizarre. Anyway, with this snapshot came an impromptu ModFest, dubbed a
+"FlashFest". The competition gave contestants 24 hours to make anything they
+could targeting this snapshot. Some of the brilliant and wildly surreal
+submissions included a "Bottle of Gender" (what), a mod that allows proposals to
+be converted to items, and then shot out of a gun (???) and a mod with a
+proposal which instils a sudden generosity into villagers, making them offer all
+trades for free (the most unrealistic one by far). The competition was a great
```

Comment:
> and a mod with a proposal which instils a sudden generosity into villagers, making them offer all trades for free (the most unrealistic one by far)

hey look it's me
(also typo "instills")

Reply:
:tiny_potato:
I'll fix it now

Reply:
wait actually @sschr15 it's instills in American English and instils in British English
so now i'm not sure which to use

Reply:
style guide when it comes

Reply:
time for another PR :p

Reply:
The new style guide (#11) now recommends British English.